### ISSUE TITLE
Config path in README incorrect for version currently in stable channel

### ISSUE THREAD

**Issue by svet-b** [dceejay/nodered.snap#22]
The README states that the settings and flow files are stored in `/var/snap/node-red/current/` (`SNAP_DATA`), which appears to be true for the release currently in the `edge` and `candidate` channels. Though the snap in the `stable` channel is older, and has those files stored under `/root/snap/node-red/current/` (`SNAP_USER_DATA`).

I know this was previously discussed in https://github.com/dceejay/nodered.snap/issues/21, and I assume that the intention is to promote the `candidate` snap to `stable` soon, which would make the README correct. But for the time being there's a disconnect between the docs and actual functionality that most people would experience.

**Comment by svet-b** [comment]
I just did a quick check re. migrating an existing `stable` install to the `candidate` version. There is a small issue with respect to migrating from `SNAP_USER_DATA` to `SNAP_DATA`. There are a few hidden files that don't get moved, and one of them contains the key for the encrypted credentials - so as it stands those would be lost. I think this is the issue that @ThyMYthOS mentioned in https://github.com/dceejay/nodered.snap/issues/21#issuecomment-821010947, described in more detail in https://discourse.nodered.org/t/moving-node-red-failed-to-decrypt-credentials/17830/15.

In practice the solution is reasonably simple...I think [line 8 of start.sh](https://github.com/dceejay/nodered.snap/blob/master/snap/local/settings/start.sh#L8) should be replaced by something like
```
mv $SNAP_USER_DATA/* $SNAP_USER_DATA/.config* $SNAP_DATA/
```
(this should work since all hidden files have names starting with `.config` - actually moving _all_ hidden files is a bit more convoluted in Bourne Shell)

**Comment by UweW** [comment]
> Closing this since the new version (that uses `$SNAP_DATA`) has been promoted to `stable`. Though from what I see there's still an issue with the migration from the old one.

you are right, after last update to version 2.2.2 node-red didn't work anymore. 
After copying .config* from /root/snap/node-red.... to /var/snap/node-red/.... it works

**Comment by svet-b** [comment]
Yeah it might be worth creating another issue to track and fix the above, as I suspect other users whose snaps auto-refresh will be hit by this situation.

**Comment by dceejay** [comment]
Pull Requests welcome, as I no longer use snaps - so have lost track of latest updates.

**Comment by svet-b** [comment]
Yup. https://github.com/dceejay/nodered.snap/pull/23
